WordPress/Classes/Stores/UserPersistentStoreFactory.swift

```diff
@@ -3,12 +3,12 @@ import Foundation
 @objc
 final class UserPersistentStoreFactory: NSObject {
     static func instance() -> UserPersistentRepository {
-        FeatureFlag.sharedUserDefaults.enabled ? UserPersistentStore.standard : UserDefaults.standard
+        FeatureFlag.contentMigration.enabled ? UserPersistentStore.standard : UserDefaults.standard
```

Comment:
Directly returning standard here will be dealt with the next PR as it is a separate issue.